domains/wik.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "RagingEnby",
+           "email": "redevstudios27@gmail.com",
+           "discord": "447861538095759365"
+        },
+    
+        "record": {
+            "A": ["34.111.254.92"]
```

Comment:
I cannot reach your website
when I go to http://34.111.254.92 i get redirected to https://34.111.254.92 which gives an SSL error